src/runtime/particle-check.ts

```diff
@@ -90,6 +91,25 @@ export class CheckIsFromHandle {
   }
 }
 
+/** A check condition of the form 'check x is from output <output>'. */
+export class CheckIsFromOutput {
+  readonly type: CheckType.IsFromOutput = CheckType.IsFromOutput;
+
+  constructor(readonly output: HandleConnectionSpec, readonly isNot: boolean) {} // ???
```

Comment:
What are the question marks for? Just delete them?

Reply:
I meant to remove them and forgot. Thanks.